.github/workflows/ci.yml

```
@@ -24,4 +24,31 @@ jobs:
 
       - uses: actions-rs/cargo@ae10961054e4aa8b4aa7dffede299aaf087aa33b
         with:
-          command: build
\ No newline at end of file
+          command: build
+
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+
+      - uses: actions-rs/toolchain@b2417cde72dcf67f306c0ae8e0828a81bf0b189f
+        with:
+          toolchain: nightly
```

keep it same as the Build ? anw #1 build fails on ci(ubuntu 20.04). will check and update here
```suggestion
          toolchain: stable
```

---

We need Nightly Rust to use `grcov` for Rust coverage.

https://github.com/mozilla/grcov#example-how-to-generate-gcda-files-for-a-rust-project

---

yes, just noticed that to build the runtime in wasm it needs `nightly` also. -> https://lib.rs/crates/substrate-wasm-builder
failure for https://github.com/DeFiCh/metachain/pull/1

---

will add a fix in #1